cmd/ably-boomer/main.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ably-forks/boomer"
+	"github.com/ably/ably-boomer/perf"
+	"github.com/ably/ably-boomer/tasks/ably"
+	"github.com/inconshreveable/log15"
+	"github.com/urfave/cli/v2"
+)
+
+var log = log15.New()
+
+func nameToEnvVars(name, prefix string) []string {
+	envVar := strings.ToUpper(name)
+	envVar = strings.ReplaceAll(envVar, "-", "_")
+	envVar = fmt.Sprintf("%s_%s", prefix, envVar)
+	return []string{envVar}
+}
+
+func generateEnvVars(prefix string, flags []cli.Flag) {
```

Comment:
slightly confusing to call this EnvVars when it's not actually an environment variable

Reply:
This function's goal is to generate environment variable names in the cli.Flags based on the flag's name. It could  be renamed `generateEnvVarNames`, that may be more precise.

Reply:
Renamed in 301ae10.

Reply:
I think my point was that they are command line options (flag names) are they not? We are removing env vars and using command line options instead, or did I misunderstand the change?

Reply:
Ah no I have used `urfave/cli` to simplify env var management, but the idea is still to use them. Command line options are also available now, but that's a side effect (and I think it makes sense to offer both options).